ApiReview/API.Polly.Core/NoDocs/Polly.Retry/RetryBackoffType.cs

```diff
@@ -0,0 +1,11 @@
+// Assembly 'Polly.Core'
+
+namespace Polly.Retry;
+
+public enum RetryBackoffType
+{
+    ExponentialWithJitter = 0,
```

Comment:
While the "best" value, I wonder if this behaviour might be surprising to a casual user compared to the (possibly naive) maybe more intuitive `Constant` behaviour for the default `0` value?

Reply:
It would also be nice having comments on these with a brief description.

Reply:
The tool that generates this API review data does provide comments, but this PR only has the version without docs (as you can see the NoDocs in the path of the files)...

Reply:
And this is the tool: https://github.com/dotnet/extensions/tree/main/eng/Tools/ApiChief

Reply:
Makes sense why the path included NoDocs. Thanks!